pkg/printers/internalversion/printers_test.go

```diff
@@ -828,12 +828,30 @@ func TestPrintNodeStatus(t *testing.T) {
 		{
 			node: api.Node{
 				ObjectMeta: metav1.ObjectMeta{
+					Name:   "foo10",
+					Labels: map[string]string{"node-role.kubernetes.io/master": "", "kubernetes.io/role": "node", "kubeadm.alpha.kubernetes.io/role": "node"},
```

Comment:
I have the same confusion. PTAL [markmaster_test.go#L52](https://github.com/kubernetes/kubernetes/blob/master/cmd/kubeadm/app/phases/markmaster/markmaster_test.go#L52). Such annotation is not the only one.

I just follow this convention here.